content/about/governance/index.en.md

```diff
@@ -16,9 +16,14 @@ And for their trouble contributing to OSRD, these companies:
 
 [^license]: As long as they don't modify the common tools without sharing their improvements. See the details of the LGPLv3 license.
 
+OSRD is currently developped by:
+
+- a team of full-time developpers at SNCF Réseau, financed by the European Union
+- unpaid individual contributors
```

Comment:
Is it necessay to specify `unpaid` here ? I find it a bit weird, also because it's no on the French version.

Reply:
it's the remnant of a previous draft, thanks for catching it